app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,8 @@
     <%= Gon::Base.render_data %>
   </head>
   <body>
+    <%= content_for :render_async %>
```

Comment:
If you flip off `allow-inline` and check out the javascript console, you'll see errors like this:

```
Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' 'unsafe-eval'". Either the 'unsafe-inline' keyword, a hash ('sha256-+PC+Kf6qxwFk9MeuRMDmsALBNWNX473zplWkiVYhJgY='), or a nonce ('nonce-...') is required to enable inline execution.
```

If those hashes have a consistent value, you can paste that `sha256-ihgwphiugwrhpiurgphiurwgphiu` value into the CSP script-src array. If they don't have consistent values, let's punt on this and we'll figure something out.